### DISCUSSION TITLE
tool/wopan/token

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#233] [Announcements]
# tool/wopan/token

import Token from 

https://alist.nn.ci/tool/wopan/token.html

**Comment by sukanka** [comment]
联通云盘获取token失败。需要输入图形验证码
```
request failed with rsp_code: 6006,rep_desc: 图形验证码未输入或缺失参数！
```

**Comment by flacce** [comment]
一样的，提示图像验证码

**Comment by pda8888** [comment]
我也一样，看来等下一版了。

**Comment by masterjian** [comment]
1. 联通网盘的验证码输入增加了输入图形的验证
2. 返回的json中，data字段已经变成加密的了，不再是标准的json结构

**Comment by jaoyun** [comment]
request failed with rsp_code: 6006,rep_desc: 图形验证码未输入或缺失参数！
